test/validators.js

```diff
@@ -4739,73 +4087,34 @@ describe('Validators', () => {
       // to be used later on for validating 'any' locale
       if (fixture.valid) allValid = allValid.concat(fixture.valid);
 
-      if (Array.isArray(fixture.locale)) {
-        test({
-          validator: 'isMobilePhone',
-          valid: fixture.valid,
-          invalid: fixture.invalid,
-          args: [fixture.locale],
-        });
-      } else {
-        test({
-          validator: 'isMobilePhone',
-          valid: fixture.valid,
-          invalid: fixture.invalid,
-          args: [fixture.locale],
-        });
-      }
+      test({
+        validator: 'isIBAN',
+        valid: fixture.valid,
+        invalid: fixture.invalid,
+        args: [fixture.locale],
+      });
     });
 
     test({
-      validator: 'isMobilePhone',
+      validator: 'isIBAN',
```

Comment:
The diff here is a little off because of the extra linting done. It should be okay if you just add the isIBAN tests without linting the file.